yarn.lock

```diff
@@ -3103,7 +3098,7 @@ ms@2.0.0:
   resolved "https://registry.yarnpkg.com/ms/-/ms-2.0.0.tgz#5608aeadfc00be6c2901df5f9861788de0d597c8"
   integrity sha1-VgiurfwAvmwpAd9fmGF4jeDVl8g=
 
-ms@2.1.2, ms@^2.1.1:
+ms@^2.1.1:
```

Comment:
Shouldn't these have been removed entirely?

Reply:
Oh weird, I'll delete yarn.lock and regenerate one. Thanks!

Reply:
@aequasi I believe is that some dependency depends on `ms`... just tried again.

Reply:
Strange!